src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx

```diff
@@ -126,20 +176,17 @@ export const TxModalWrapper = ({
   })
   if (safeTxGas == null) safeTxGas = safeTxGasEstimation
 
-  const { gasCostFormatted, gasPriceFormatted, gasMaxPrioFeeFormatted, gasLimit, txEstimationExecutionStatus } =
-    useEstimateTransactionGas({
-      txData,
-      txRecipient: txTo || safeAddress,
-      txConfirmations,
-      txAmount: txValue,
-      safeTxGas: safeTxGas || manualSafeTxGas,
-      manualGasPrice,
-      manualMaxPrioFee,
-      manualGasLimit,
-      operation,
-      isExecution: doExecute,
-      approvalAndExecution,
-    })
+  const { gasPriceFormatted, gasPrice, gasMaxPrioFee, gasMaxPrioFeeFormatted } = useEstimateTransactionGas({
+    manualGasPrice,
+    manualMaxPrioFee,
+    isExecution: doExecute,
+    txData,
+  })
+
+  const getGasCostFormatted = useCallback(() => {
+    if (!gasLimit) return '> 0.001'
```

Comment:
If the `gasLimit` estimation fails we catch the error in the `useEstimateGasLimit`hook and return `'0'`. I think we have to additionally test for `gasLimit === '0'`here

Reply:
Good catch!